lab-davidw/test/client-test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const expect = require('chai').expect;
+const testClient = require('../model/client.js');
```

Comment:
When requiring in a module that is a constructor function, you need to capitalize it per convention, i.e.:

```
const TestClient = require('../model/client.js');

```
No points lost for this round, but I'll likely mark it off if I see it again in future labs.